src/pipelines/epidemiology/config.yaml

```
@@ -609,6 +609,16 @@ sources:
       job_group: all
       deferred: true
 
+  # Data sources for ID level 2
+  - name: pipelines.epidemiology.id_authority.IndonesiaLevel2DataSource
+    fetch:
+      - url: "https://andrafarm.com/api/covid19/kota/{}"
+    test:
+      location_key_match: 'IDX.*'
```

This regex will not match the location keys, it probably should be:
```suggestion
      location_key_match: '^ID_[^_]+[^_]+$'
```

---

done. will change this once level1 data is added